public/Sidebar/sidebar.js

```diff
@@ -0,0 +1,50 @@
+var entire = document.getElementById("f");
+
+entire.innerHTML=`
+
+<button onclick="frontpage()" id="f_button"><i id="right" class="fa fa-long-arrow-right "></i></button>
+<div id="f_title" ></div>
+<button onclick="backpage()" id="b_button"><i id="left" class="fa fa-long-arrow-left "></i></button>
+<div id="b_title"></div>
+
+`;
+
+let f_b = document.getElementById("f_button");
+let b_b = document.getElementById("b_button");
+document.body.appendChild(f_b);
+document.body.appendChild(b_b);
+
+
+$(document).ready(function() {
+    $('#f_title').text($("a.mt-icon-next-article span").first().text());
```

Comment:
Could you please move all your code into this function **_or_** wrap everythin a self-executing anonymous function? This prevents global variable pollution and modularizes your code so that it is easier to handle/debug.